truss/templates/base.Dockerfile.jinja

```diff
@@ -37,6 +37,10 @@ RUN apt-get update && apt-get install --yes --no-install-recommends $(cat system
 
 
 {% block install_requirements %}
+    {%- if should_install_user_requirements_file %}
+COPY ./user_requirements.txt user_requirements.txt
+RUN pip install -r user_requirements.txt --no-cache-dir && rm -rf /root/.cache/pip
```

Comment:
nit -- there are a couple requirements.txt files in here now, it would be nice if "user_requirements.txt" could be templated it (i think will help w/ readability).